dask_cloudprovider/aws/ec2.py

```diff
@@ -356,6 +358,10 @@ class EC2Cluster(VMCluster):
         Whether to use a private IP (if True) or public IP (if False).
 
         Default ``False``.
+    enable_detailed_monitoring: bool (optional)
+        Whether to enable detailed monitoring for created instances.
+        See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-cloudwatch-new.html
+        Default ``None``.
```

Comment:
The docstring here should reflect the actual default, so from the config this would be ``False``.